aws/csm/metricChan.go

```diff
@@ -0,0 +1,54 @@
+package csm
+
+import (
+	"sync/atomic"
+)
+
+const (
+	runningEnum = iota
+	pausedEnum
+)
+
+var (
+	// DefaultChannelSize ...
+	DefaultChannelSize = 100
```

Comment:
This value definition should live in the reporter since that is where it is used.  Also why a var instead of const?

Reply:
Yea, want to be changeable by the user. Probably should remove `Default` from the name.